typed-racket-doc/typed-racket/scribblings/guide/more.scrbl

```diff
@@ -89,13 +89,21 @@ of @racket[Number]s.
 @racketblock[(lambda ([x : String] (unsyntax @tt["."]) [y : Number #,**]) (apply + y))]
 
 This function has the type @racket[(-> String Number #,** Number)].
+To specify the return type, add a type annotation after the arguments:
+
+@racketblock[(lambda ([x : String] (unsyntax @tt["."]) [y : Number #,**]) : (U Number String) (apply + y))]
+
 Functions defined by cases may also be annotated:
 
 @racketblock[(case-lambda [() 0]
                           [([x : Number]) x])]
 
 This function has the type
 @racket[(case-> (-> Number) (-> Number Number))].
+To specify the return type, either annotate the entire function or use the
+@seclink["Annotating_Expressions" #:doc '(lib "typed-racket/scribblings/ts-guide.scrbl")]{expression annotation form}
```

Comment:
I think `#:doc` can be elided here, since this current Scribble file is already within `typed-racket/scribblings/ts-guide.scrbl`?